src/Findologic/Response/Xml21/Filter/Values/FilterValue.php

```diff
@@ -54,7 +54,7 @@ public function getTranslated(): TranslatedName
         return $this->translated;
     }
 
-    public function getUuid(): string
+    public function getUuid(): ?string
```

Comment:
When could this be `null`?

Reply:
As of now, it is null for categories filter.